pwndbg/disasm/__init__.py

```diff
@@ -100,11 +100,13 @@ def get_one_instruction(address):
         pwndbg.disasm.arch.DisassemblyAssistant.enhance(ins)
         return ins
 
-def one(address=None):
+
+
+def one(address='unset_argument'):
+    if address == 'unset_argument':
```

Comment:
```
def one(address=None):
    if address is None or not pwndbg.memory.peek(address):
        return None
    if address is None:
        address = pwndbg.regs.pc
```

that doesn't make any sense as "address = pwndbg.regs.pc" will never be assigned. therefor if you invoke one() without an argument it always returns None. however try removing the early exit yourself and step throug a program, it will screw up all the disassembly as other calls to one(var) having passed var=None because of some reasons this will scrwe up everything.